sdk/react/ooyalaSkinBridgeListener.js

```diff
@@ -76,7 +76,7 @@ OoyalaSkinBridgeListener.prototype.onSeekStarted = function(e) {
 OoyalaSkinBridgeListener.prototype.onSeekComplete = function(e) {
   Log.log( "onSeekComplete");
   this.skin.setState({
-    playhead: e.playhead,
```

Comment:
re-named seekInfo variables for better understanding of them. at seekStart event, playhead should be at seekStart point and over seekComplete event, playhead should be the seekEnd point.